README.md

```diff
@@ -1559,102 +2620,167 @@ The route53 types set up various types of Route53 records:
 
 For public zones, validated but not used.
 
 #### Type: s3_bucket
 
 #####`name`
-*Required* The name of the bucket to managed.
+
+Required.
+
+The name of the bucket to managed.
 
 #####`policy`
+
 A JSON parsable string of the policy to apply to the bucket.
 
 #### Type: sqs_queue
 #####`name`
-*Required* The name of the SQS queue.
+
+Required.
+
+The name of the SQS queue.
 
 #####`region`
-*Required* The region in which to create the SQS Queue. For valid values, see [AWS Regions](http://docs.aws.amazon.com/general/latest/gr/rande.html#ec2_region).
+
+Required.
+
+The region in which to create the SQS Queue. 
+
+Valid values are:
+
+See [AWS Regions](http://docs.aws.amazon.com/general/latest/gr/rande.html#ec2_region).
 
 #####`delay_seconds`
-*Optional* The time in seconds that the delivery of all messages in the queue will be delayed. Default value: 0
+
+Optional.
+
+The time in seconds that the delivery of all messages in the queue will be delayed. 
+
+Default value: `0`.
 
 #####`message_retention_period`
-*Optional* The number of seconds Amazon SQS retains a message. Default value: 345600
+
+Optional.
+
+The number of seconds Amazon SQS retains a message. 
+
+Default value: `345600`.
 
 #####`maximum_message_size`
-*Optional* The limit of how many bytes a message can contain before Amazon SQS rejects it.
+
+Optional.
+
+The limit of how many bytes a message can contain before Amazon SQS rejects it.
 
 #####`visibility_timeout`
-*Optional* The number of seconds during which Amazon SQS prevents other consuming components from receiving and processing a message. Default value: 30
 
+Optional.
+
+The number of seconds during which Amazon SQS prevents other consuming components from receiving and processing a message. 
+
+Default value: `30`.
 
 ##Limitations
 
-This module requires Ruby 1.9 or later and is only tested on Puppet
-versions 3.4 and later.
+This module requires Ruby 1.9 or later and is only tested on Puppet versions 3.4 and later.
+
+At the moment this module only supports a few of the resources in the AWS API. These resources also exist a bit outside the normal host level resources like `package`, `file`, `user`, etc. 
 
-At the moment this module only supports a few of the resources
-in the AWS API. These resources also exist a bit outside the
-normal host level resources like `package`, `file`, `user`, etc. We're
-really interested to see how people use these new resources, and what
-else you would like to be able to do with the module.
+We're really interested to see how people use these new resources, and what else you would like to be able to do with the module.
 
-Note that this module also requires at least Ruby 1.9 and is only tested on Puppet
-versions from 3.4. If this is too limiting please let us know.
+Note that this module also requires at least Ruby 1.9 and is only tested on Puppet versions from 3.4. If this is too limiting please let us know.
```

Comment:
I'd take out that last line 'if this is too limiting', it's just what we support.